PEP 622: Strengthen the argument for _ as wildcard (#1488)
python/peps: pep-0622.rst

the bridge.)

Another proposal was to use ``?``.  This could be acceptable, although
it would require modifying the tokenizer.

Also, ``_`` is already used
as a throwaway target in other contexts, and this use is pretty
similar.  This example is from ``difflib.py`` in the stdlib::

  for tag, _, _, j1, j2 in group: ...

Perhaps the most convincing argument is that ``_`` is used as the
wildcard in every other language we've looked at supporting pattern
matching: C#, Elixir, Erlang, F#, Haskell, Mathematica, OCaml, Ruby,
Rust, Scala, and Swift.  Now, in general, we should not be concerned
too much with what another language does, since Python is clearly
different from all these languages.  However, if there is such an
overwhelming and strong consensus, Python should not go out of its way
to do something completely different -- particularly given that ``_``

> works well in Python is already in use as a throwaway target.

Reviewer: ```suggestion
works well in Python and is already in use as a throwaway target.
```

> Note that ``_`` should not be assigned to by patterns -- this avoids

Reviewer: ```suggestion
Note that ``_`` is not assigned to by patterns -- this avoids
```
